### PR TITLE
[FLINK-5431] Add configurable timePattern for client akka status

### PR DESCRIPTION
**Pull request by tonycox**
Thanks for contributing to Apache Flink. Before you open your pull request, please take the following check list into consideration.
If your changes take all of the items into account, feel free to open your pull request. For more information and/or questions please refer to the [How To Contribute guide](http://flink.apache.org/how-to-contribute.html).
In addition to going through the list, please provide a meaningful description of your changes.

- [ ] General
  - The pull request references the related JIRA issue ("[FLINK-XXX] Jira title text")
  - The pull request addresses only one issue
  - Each commit in the PR has a meaningful commit message (including the JIRA id)

- [ ] Documentation
  - Documentation has been added for new functionality
  - Old documentation affected by the pull request has been updated
  - JavaDoc for public methods has been added

- [ ] Tests & Build
  - Functionality added by the pull request is covered by tests
  - `mvn clean verify` has been executed successfully locally or a Travis build has passed

### CODE REVIEW

File: docs/setup/config.md [hunk 1]
```diff
@@ -151,6 +151,8 @@ one of the values specified in `security.kerberos.login.contexts`.
 
 ### Other
 
+- `client.logging.time-pattern`: (DEFAULT: `yyyy-MM-dd'T'HH:mm:ss`) The date time pattern applied to client akctor logger.
```
Comment (zentol) [inline docs/setup/config.md]: `acktor` -> `actor`

**Comment by tonycox** [comment]
@zentol What do you think about this PR ?

**Comment by StephanEwen** [comment]
Not sure about this. This seems to implement in Flink what is actually the responsibility of the logger.

How about just updating the client such that there are two loggers configured: 
  1. The one that there is now (all internal stuff)
  2. The one that is used for status updates.

The status logger goes to file and console. You can customize the console logging in the logger config then.

That seems a better separation of concerns, and more generally extensible.

**Comment by StephanEwen** [comment]
@tonycox What do you think about my suggestion? If you agree, could you close this pull request?

**Pull request closed without merging**